keploy.sh

```diff
@@ -96,21 +110,59 @@ installKeploy (){
         fi
     }
 
+    check_docker_status_for_linux() {
+        check_sudo
+        sudoCheck=$?
+        network_alias=""
+        if [ "$sudoCheck" -eq 0 ] && [ $OS_NAME = "Linux" ]; then
+            # Add sudo to docker
+            network_alias="sudo"
+        fi
+        if ! $network_alias which docker &> /dev/null; then
+            echo -n "Docker not found on device, please install docker and reinstall keploy if you are willing to use applications with docker"
```

Comment:
The logs for mac and linux shouldn't be so different in my opinion. and I think "willing" doesn't fit well in a log message. I think just change both to "Docker not found on device. Please install docker and reinstall keploy". Same thing for starting docker.

Reply:
we shouldn't force user to use docker on linux if he has no application with docker